CHANGELOG.md

```diff
@@ -6,14 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Changed
+- **BREAKING**: The addNewKeyring method now expects an object containing the property `privateKeys` of type `string[]` in case the supplied keyring is a "Simple Keyring". ([#202](https://github.com/MetaMask/KeyringController/pull/202))
+- docs: Update README ([#209](https://github.com/MetaMask/KeyringController/pull/209))
+- deps: Unpin and bump @metamask/eth-sig-utils@5.0.2->^5.1.0 ([#224](https://github.com/MetaMask/KeyringController/pull/224))
```

Comment:
Do we want to remove these last two entries since they aren't user-facing?

Reply:
This entry is; this lets users of this package use later versions of eth-sig-util. Agreed on the readme update though

Reply:
Oh, fair.